src/topbar/components/KTopbarUI.vue

```diff
@@ -11,7 +11,10 @@
         :src="currentGroup.photoUrls.thumbnail"
         style="height: 95%"
       >
-      <KarrotLogo v-else />
+      <KarrotLogo
+        v-else
+        :show-loading="true"
```

Comment:
Just `show-loading` should be enough!

Reply:
Ah yeah, I wasn't sure if that suddenly made it a directive... will update :)